packages/node_modules/@ciscospark/react-component-utils/src/constants.js

```diff
@@ -68,3 +68,12 @@ export const SPACE_TYPE_ONE_ON_ONE = 'direct';
 export const SPACE_TYPE_GROUP = 'group';
 
 export const TAG_MUTED = 'MUTED';
+
+export const attachmentType = 'AdaptiveCard';
+export const cardReference = 'cardReference';
+export const cards = 'cards';
+
+export const DEVELOPER = 'developer';
```

Comment:
```suggestion
export const FEATURES_DEVELOPER = 'developer';
```
Would be better and more consistent with the other feature constants.